chore(ddx): content review dedibox (#3495)
scaleway/docs-content: dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx

---
meta:
  title: How to configure the network of a virtual machine on a Dedibox host

>   description: Learn how to configure network settings for virtual machines on various operating systems including Ubuntu, Debian, CentOS, and Windows Server. 

Reviewer: ```suggestion
  description: Learn how to configure network settings for virtual machines on various operating systems, including Ubuntu, Debian, CentOS, and Windows Server. 
```

content:
  h1: How to configure the network of a virtual machine on a Dedibox host

>   paragraph: Learn how to configure network settings for virtual machines on various operating systems including Ubuntu, Debian, CentOS, and Windows Server. 

Reviewer: ```suggestion
  paragraph: Learn how to configure network settings for virtual machines on various operating systems, including Ubuntu, Debian, CentOS, and Windows Server. 
```

            addresses: [fail.over.ip.address/32]
            gateway4: 62.210.0.1
            nameservers:

>               addresses: [ "51.159.47.28", "51.159.47.26" ]

Author: ```suggestion
              addresses: [ "51.159.47.28", "51.159.47.26" ] # Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
```

3. Save the file and exit the text editor.
4. Set the DNS server information by editing the file `/etc/resolv.conf`:
    ```
    sudo nano /etc/resolv.conf
    ```
5. Set the DNS resolver by adding the following lines to the file. Then save it and exit the text editor:
    ```
    nameserver 51.159.47.28
    nameserver 51.159.47.26

>     ```

Author: ```suggestion
    ```
    <Message type="tip">
        Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
    </Message>
```

3. Save the file and exit the text editor.
4. Set the DNS server information by editing the file `/etc/resolv.conf`:
    ```
    sudo nano /etc/resolv.conf
    ```
5. Set the DNS resolver by adding the following lines to the file. Then save it and exit the text editor:
    ```
    nameserver 51.159.47.28
    nameserver 51.159.47.26
    ```
    <Message type="tip">

>       Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)

Reviewer: ```suggestion
      Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performance. [Check the IPs for each datacenter](/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
```

    NETMASK=255.255.255.255
    IPADDR=fail.over.ip.address
    GATEWAY=62.210.0.1

>     DNS1=51.159.47.28 # Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)

Reviewer: Is this really supposed to be formatted like this?

    * DNS 1: `51.159.47.28`
    * DNS 2: `51.159.47.26`

    <Message type="tip">

>       Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)

Reviewer: ```suggestion
      Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performance. [Check the IPs for each datacenter](/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
```
